repo-scripts/api-documenter/src/cli/MarkdownAction.ts

```diff
@@ -33,10 +35,24 @@ export class MarkdownAction extends BaseAction {
     });
   }
 
+  protected onDefineParameters(): void {
+    super.onDefineParameters();
+
+    this._sortFunctions = this.defineStringParameter({
+      parameterLongName: '--sort-functions',
+      argumentName: 'PRIORITY_PARAMS',
+      description:
+        `Sorts functions tables and listings by first parameter.` +
+        ` Provide comma-separated strings for preferred params to be ` +
```

Comment:
not required but I would remove the space before here and add it onto the end of the line above.